ref/feature/openidConnectServer-1.0.adoc

```diff
@@ -1,13 +1,15 @@
 = OpenID Connect Provider 1.0
-:linkcss: 
+:linkcss:
 :page-layout: feature
-:nofooter: 
+:nofooter:
 
 include::./gen/openidConnectServer-1.0.gendoc[tag=description]
 
+include::./openidConnectServer/description.adoc[]
+
 include::./gen/openidConnectServer-1.0.gendoc[tag=enable]
 
-include::./gen/openidConnectServer-1.0.gendoc[tag=config]
```

Comment:
It looks like you might've removed `include::./gen/openidConnectServer-1.0.gendoc[tag=config]`, which I don't think should be removed. See https://raw.githubusercontent.com/OpenLiberty/docs/draft/ref/feature/socialLogin-1.0.adoc for an example of where to include the description and examples files.